templates/zerver/search_operators.html

```diff
@@ -94,15 +94,14 @@
         </p>
 
         <p>
-            would search for messages sent by
+            {% trans %}would search for messages sent by{% endtrans %}
             <span class="operator_value">user@example.com</span>
-            to stream
+            {% trans %}to stream{% endtrans %}
             <span class="operator_value">streamname</span>
-            containing the keyword
+            {% trans %}containing the keyword{% endtrans %}
```

Comment:
so, I think you can't atomize some of these pieces, because words go in another order in ohter languages.    So I think this paragraph might still need to be tagged for translation.